website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -628,7 +628,7 @@ various namespaces is also cached to speed up future operations.
 
 The `cluster-user-auth` and `oidc-auth` secrets provide information for authenticating
 to the application. The former holds the username and bcrypt-hashed password
-for the [emergency user](../../configuration/emergency-user.mdx), and the latter holds OIDC configuration.
+for the [emergency user](#configuring-the-emergency-user), and the latter holds OIDC configuration.
```

Comment:
same here

Reply:
can you add a screenshot of the section from the live docs? i can't see it at all 🙈

Reply:
I did in the last comment -- wild!